### PR TITLE
Added a quick fix to #36

### PR DESCRIPTION
**Pull request by adminq80**
I added a `try` `catch` block to check if either `privateKey` or `publicKey` don't have toString method attach to them. This would give us an indication that `openssl` command failed to execute.

### CODE REVIEW

File: src/algorithms/ECDSA.js [hunk 1]
```diff
@@ -163,8 +163,12 @@ class ECDSA extends Algorithm {
         // TODO may need to remove -noout if ec params is needed
         let privateKey = spawnSync('openssl', ['ecparam','-name',osslCurveName.name,'-genkey','-noout']).stdout
         let publicKey = spawnSync('openssl', ['ec', '-pubout'], { input: privateKey }).stdout
+        try{
```
Comment (EternalDeiwos) [inline src/algorithms/ECDSA.js]: should be `try {`

File: src/algorithms/ECDSA.js [hunk 2]
```diff
@@ -163,8 +163,12 @@ class ECDSA extends Algorithm {
...
         keypair.privateKey = privateKey.toString('ascii').trim()
```
Comment (EternalDeiwos) [inline src/algorithms/ECDSA.js]: indentation

File: src/algorithms/ECDSA.js [hunk 3]
```diff
@@ -163,8 +163,12 @@ class ECDSA extends Algorithm {
...
         keypair.publicKey = publicKey.toString('ascii').trim()
+        } catch(error){
```
Comment (EternalDeiwos) [inline src/algorithms/ECDSA.js]: should be `} catch (error) {`

File: src/algorithms/RSASSA-PKCS1-v1_5.js [hunk 4]
```diff
@@ -143,9 +143,12 @@ class RSASSA_PKCS1_v1_5 extends Algorithm {
       let privateKey = spawnSync('openssl', ['genrsa', modulusLength || 4096]).stdout
       let publicKey = spawnSync('openssl', ['rsa', '-pubout'], { input: privateKey }).stdout
 
+	  try{
```
Comment (EternalDeiwos) [inline src/algorithms/RSASSA-PKCS1-v1_5.js]: indentation and spacing (see above)

File: src/algorithms/RSASSA-PKCS1-v1_5.js [hunk 5]
```diff
@@ -143,9 +143,12 @@ class RSASSA_PKCS1_v1_5 extends Algorithm {
...
       keypair.privateKey = privateKey.toString('ascii')
       keypair.publicKey = publicKey.toString('ascii')
-
+	  } catch (error){
```
Comment (EternalDeiwos) [inline src/algorithms/RSASSA-PKCS1-v1_5.js]: indentation and spacing

**Comment by adminq80** [comment]
Thanks for your comments, and sorry I didn't review how to contribute to the project before sending in the PR. I hope these commits qualify to be merged. Please let me know if there are any problems with this PR.

**Comment by adminq80** [comment]
Thanks for accepting the PR

**Pull request merged**